src/app/models/__tests__/workspace.server.model.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { ObjectId } from 'bson-ext'
+import { omit } from 'lodash'
+import mongoose from 'mongoose'
+
+import { getWorkspaceModel } from 'src/app/models/workspace.server.model'
+import { IUserSchema } from 'src/types'
+
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+
+const Workspace = getWorkspaceModel(mongoose)
+
+const MOCK_USER_ID = new ObjectId()
+const MOCK_FORM_ID = new ObjectId()
+const MOCK_WORKSPACE_ID = new ObjectId()
+const MOCK_WORKSPACE_FIELDS = {
+  _id: MOCK_WORKSPACE_ID,
+  title: 'Workspace1',
+  admin: MOCK_USER_ID,
+  formIds: [],
+}
+
+describe('Workspace Model', () => {
```

Comment:
do we need tests on the format of the title as well? normally the way I think about model-level tests is that we should test the validation on each key

Reply:
Okay, will add the tests for title validation as well